src/it/java/com/google/checkstyle/test/base/AbstractIndentationTestSupport.java

```diff
@@ -60,6 +60,16 @@
         return getLinesWithWarnAndCheckComments(fileName, TAB_WIDTH);
     }
 
+    /**
+     * Returns line numbers for lines with Warn comments.
+     *
+     * @param aFileName file name.
+     * @param tabWidth tab width (passed value is 4 to this method).
+     * @return array of line numbers containing warn comments ("warn").
```

Comment:
containing 'warn' comments.

Let's make consistent usage of `'warn'` or `"warn"`

Reply:
Done!